src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -544,9 +805,16 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
     }
 
     ov::Tensor get_inputs_embeds_llava(const std::string& prompt, const std::vector<ov::Tensor>& images) {
-        std::string image_token = "<image>"; // TODO Consider getting from vlm_config or json
-        std::string formatted_prompt = "USER: " + (images.empty() ? prompt : image_token + "\n" + prompt) + " ASSISTANT:";
-        ov::Tensor input_ids = m_tokenizer.encode(formatted_prompt).input_ids;
+        std::string image_token = m_vlm_config.im_start;
+        std::string formatted_prompt = images.empty() ? prompt : image_token + "\n" + prompt;
+        
+        // std::string chat_template_fallback = m_templated_chat_history + " USER: " + formatted_prompt + " ASSISTANT: ";
+        // chat_template_fallback = chat_template_fallback.erase(0, chat_template_fallback.find_first_not_of(' '));
+        
+        // Adapted from llava-1.5-7b-hf chat_template.json
+        std::string chat_template_fallback = "{% for message in messages %}{% if message['role'] == 'user' %}{{ 'USER: ' + message['content'] + ' ' }}{% else %}{{ 'ASSISTANT: ' + message['content'] + ' ' }}{% endif %}{% endfor %}{% if add_generation_prompt %}{{ 'ASSISTANT:' }}{% endif %}";
```

Comment:
@Wovchena @ilya-lavrenov Which option do you think is better - concatenating prompt from strings (commented) or using an adapted/simplified jinja chat template?

Reply:
Concatenating prompt from strings is more precise approach, right? What's the difference between the original and simplified chat template?

Reply:
I would say concatenating prompt from strings is more readable and compact. They both render the same templated history string.

The original chat template from `chat_template.json` has some constructions unsupported by Jinja2Cpp (e.g. `.upper()`, `| selectattr('type', 'equalto', 'image')`, `{% generation %}{{ content['text'] + ' '}}{% endgeneration %}`). 
Also, image tokens are added within original chat template logic, while we are preparing prompt with <image> tokens by ourselves.

Reply:
I think the problem here is that we allow overriding outside chat_template. Applying it manually would ignore new template.

I don't think it's a big deal. Just describe it in docstrings.